defaults/main.yml

```diff
@@ -5,25 +5,58 @@ apache_listen_port_ssl: 443
 
 apache_vhost_name: "owncloud.conf"
 
-# Additional properties:
-# 'serveradmin, serveralias, allow_override, options, extra_parameters'.
+# @var apache_vhosts: >
 # apache_vhosts:
 #   - servername: "cloud.owncloud.demo"
 #     documentroot: "/var/www/owncloud"
+# @end
+
+# @var apache_vhosts:example: >
+# apache_vhosts:
+#   - servername: "cloud.owncloud.demo"
+#     documentroot: "/var/www/owncloud"
+#
+#     # optional properties
+#     serveradmin:
+#     serveralias:
+#     allow_override: defaults to apache_allow_override
+#     options: defaults to apache_options
+#     extra_parameters: []
+# @end
 
 apache_vhosts_ssl: []
+# @var apache_vhosts_ssl:description: >
+# Define SSL enbaled Vhost configurations. The example below shows all available properties that can be set.
+# @end
+
 # @var apache_vhosts_ssl:example: >
 # apache_vhosts_ssl:
-#   - servername: "local.dev"
+#   - servername: "cloud.owncloud.demo"
 #     documentroot: "/var/www/html"
+#
+#     # Location of the SSL certificate file to use. The file has to exist on the target host already or
+#     # you can use `certificate_file_source` to deploy a certificate located on the Ansible control host.
 #     certificate_file: "/path/to/certificate.crt"
-#     certificate_file_source: "/path/to/source/on/ansible_host/certificate.crt"
+#
+#     # Location of the SSL certificate key file to use. The file has to exist on the target host already or
+#     # you can use `certificate_key_source` to deploy a certificate key located on the Ansible control host.
 #     certificate_key_file: "/path/to/certificate.key"
+#
+#     # optional properties
+#     serveradmin:
+#     serveralias:
+#     allow_override: defaults to apache_allow_override
+#     options: defaults to apache_options
+#     extra_parameters: []
+#
+#     # The properties `certificate_file_source` and `certificate_key_source` can be used to deploy SSl
+#     # certificate files located on the Ansible control host to the target host (web server). If these
+#     # variables are not set, the SSL certificates need to be located on the tarhet host already.
+#     certificate_file_source: "/path/to/source/on/ansible_host/certificate.crt"
 #     certificate_key_source: "/path/to/source/on/ansible_host/certificate.key"
-#     # Optional.
 #     certificate_chain_file: "/path/to/certificate_chain.crt"
```

Comment:
See: https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslcertificatefile
[SSLCertificateChainFile](https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslcertificatechainfile) is obsolete since SSLCertificateFile can load intermediate and root CA itself.
In anyway having both running in parallel, like we have here: 
https://github.com/owncloud-ansible/apache/blob/1da034fec9b7850a40e3387717f49a168d5a93a1/templates/owncloud.conf.j2#L51
could lead to problems, if somebody loads a SSLCertificateFile and a SSLCertificateChainFile including the server cert. I guess we should just drop this option completely?
But watch out, will be a BREAKING change.

in the documentation, we need to explain then, that SSLCertFile can hold either cert only + giving a CA file path or the full certificate chain up to root cert.